sbnobj/Common/CRT/classes_def.xml

```diff
@@ -159,4 +160,23 @@
   <!--<class name="art::Assns<recob::OpFlash,sbn::crt::CRTHit,sbn::crt::CRTPMTMatchingInfo>"/>
  <class name="art::Wrapper<art::Assns<recob::OpFlash,sbn::crt::CRTHit,sbn::crt::CRTPMTMatchingInfo> >"/>
  <class name="art::Wrapper<art::Assns<sbn::crt::CRTHit,recob::OpFlash,sbn::crt::CRTPMTMatchingInfo> >"/>-->
+
+  <!-- CRTT0Tagging Information -->
+<enum name="sbn::crt::CRTTaggingMethod"/>
+<enum name="sbn::crt::CRTTaggingTrackFit"/>
+<class name="sbn::crt::CRTT0TaggingInfo"/>
```

Comment:
This `sbn::crt::CRTT0TaggingInfo` needs a `ClassVersion="10"` to enable ROOT I/O schema evolution (see also other objects here, like `sbn::crt::CRTTrack`).